sqlmesh/dbt/source.py

```diff
@@ -28,11 +28,9 @@ class SourceConfig(GeneralConfig):
         columns: Columns within the source
     """
 
-    # sqlmesh fields
-    config_name: str = ""
-
     # DBT configuration fields
-    name: t.Optional[str] = None
+    name: str = ""
+    source_name_: str = Field("", alias="source_name")
```

Comment:
Do we even need the `source_name` property name now? Overloading this property seems weird, given the method returns something entirely different.

Reply:
The source_name property is meant to represent the name within the db. I thought about changing model_name and source_name to model_sql_name and source_sql_name. What are your thoughts?

Reply:
Can we just do sql_name? That should help I think